autoPyTorch/pipeline/components/training/trainer/base_trainer_choice.py

```diff
@@ -324,10 +401,20 @@ def _fit(self, X: Dict[str, Any], y: Any = None, **kwargs: Any) -> 'TrainerChoic
             if 'cuda' in X['device']:
                 torch.cuda.empty_cache()
 
+        if self.choice.use_stochastic_weight_averaging and self.choice.swa_updated:
+            # update batch norm statistics
+            swa_utils.update_bn(X['train_data_loader'], self.choice.swa_model.double())
```

Comment:
It is better to use keyword arguments, because we cannot easily track what object it is.